.vscode/launch.json

```diff
@@ -11,6 +11,36 @@
       "program": "${workspaceFolder}/dist/generator.js",
       "preLaunchTask": "tsc: build - tsconfig.json",
       "outFiles": ["${workspaceFolder}/dist/**/*.js"]
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Jest Current File",
+      "program": "${workspaceRoot}/node_modules/jest/bin/jest.js",
+      "args": [
+        "${relativeFile}",
+        "--config",
+        "${workspaceRoot}/jest.config.js",
+        "--runInBand"
+      ],
+      "disableOptimisticBPs": true,
+      "console": "integratedTerminal"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Jest Highlighted Test",
+      "program": "${workspaceRoot}/node_modules/jest/bin/jest.js",
+      "args": [
+        "${relativeFile}",
+        "--testNamePattern",
+        "${selectedText}",
```

Comment:
TIL

Reply:
Spent enough time debugging unit tests in ```nexus-prisma``` that it seemed worth it to add the script I use in my repo :smile: